frontend/src/components/cluster/Overview.tsx

```diff
@@ -75,7 +75,7 @@ const useStyles = makeStyles(theme => ({
 
 function EventsSection() {
   const classes = useStyles();
-  const filterFunc = useFilterFunc();
+  const filterFunc = useFilterFunc(false);
```

Comment:
I understand why you added this patch, but I release now that we have a fundamental mistake in the events' page: it is missing a namespace column. I will open an issue.